invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -469,25 +474,7 @@ class CommunityProfileForm extends Component {
                             "metadata.organizations",
                             []
                           )}
-                          serializeSuggestions={(organizations) =>
-                            _map(organizations, (organization) => {
-                              // eslint-disable-next-line no-prototype-builtins
-                              const isKnownOrg = this.knownOrganizations.hasOwnProperty(
-                                organization.name
-                              );
-                              if (!isKnownOrg) {
-                                this.knownOrganizations = {
-                                  ...this.knownOrganizations,
-                                  [organization.name]: organization.id,
-                                };
-                              }
-                              return {
-                                text: organization.name,
-                                value: organization.name,
-                                key: organization.name,
-                              };
-                            })
-                          }
+                          serializeSuggestions={this.serializeAffiliations}
```

Comment:
nit: also possible without declaring a new method
```suggestion
                          serializeSuggestions={(organizations)=>AffiliationsSuggestions(organizations, true, false)}
```